src/shared/lib/OncoKbUtils.spec.tsx

```diff
@@ -1,20 +1,20 @@
-import * as SortUtils from './SortUtils';
+import * as OncoKbUtils from './OncoKbUtils';
```

Comment:
I think this is a rebase singularity :)

Actually `SortUtils.spec.jsx` was renamed to `SortUtils.spec.tsx`, and a new (empty) `OncoKbUtils.spec.tsx` was added.